client/components/editor/structure/Assessments.vue

```diff
@@ -32,7 +32,7 @@
 <script>
 import { mapActions, mapGetters, mapMutations } from 'vuex';
 import AddElement from 'tce-core/AddElement';
-import AssessmentItem from './AssessmentItem';
+import AssessmentItem from 'tce-core/AssessmentItem';
```

Comment:
Shouldn't this file be deleted entirely 🤔 ?

Reply:
If you are talking about the `Assessments` file then yes. I missed that. If you are talking about the `AssessmentItem`, it is used by both the Exam custom container and this container. And I am not sure if we want to duplicate that. The Exam custom container will probably never be used again so maybe we can remove it as well at some point and then remove `AssessmentItem` from `tce-core`.

Reply:
The first one 😉